sui-execution/latest/sui-adapter/src/error.rs

```diff
@@ -30,6 +31,11 @@ pub(crate) fn convert_vm_error<S: MoveResolver<Err = SuiError>>(
             ExecutionFailureStatus::VMInvariantViolation
         }
         (StatusCode::ABORTED, Some(code), Location::Module(id)) => {
+            let abort_location_id = if resolve_abort_location_to_package_id {
+                state_view.relocate(id).unwrap_or(id.clone())
```

Comment:
`unwrap_or_else`? 
What's more expensive? the string or the lambda? lol

Reply:
I have no idea lol, but happy to make it an unwrap_or_else :)